packages/integration_test/android/build.gradle

```diff
@@ -1,64 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-group 'dev.flutter.plugins.integration_test'
-version '1.0-SNAPSHOT'
-
-buildscript {
-    repositories {
-        google()
-        mavenCentral()
-    }
-
-    dependencies {
-        classpath 'com.android.tools.build:gradle:7.3.1'
-    }
-}
-
-rootProject.allprojects {
-    repositories {
-        google()
-        mavenCentral()
-    }
-}
-
-// project.getTasks().withType(JavaCompile){
-//     options.compilerArgs << "-Xlint:all" << "-Werror"
-// }
-
-apply plugin: 'com.android.library'
-
-android {
-    // Conditional for compatibility with AGP <4.2.
-    if (project.android.hasProperty("namespace")) {
-        namespace 'dev.flutter.integration_test'
-    }
```

Comment:
PR #142000 will start enforcing Gradle 7, so this checks won't make sense anymore

Reply:
Heads up that the linked PR actually won't start enforcing anything until a full release later. I'm starting with only "warning" versions, so when it lands we will start warning that anything <7 _will soon_ be unsupported

Reply:
ah okay, thanks! So I'll make sure we support older Gradle/AGP as well